README.md

```diff
@@ -1,24 +1,154 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 [Budget App] <a name="about-project"></a>
 
-* Database creation
+> Describe your project in 1 or 2 sentences.
 
-* Database initialization
+**[Budget App]** A budget app is a tool designed to help individuals or households manage their finances by tracking income and expenses. With features like expense categorization, and expenses tracking, budget apps can be a useful way to improve financial management and achieve financial goals.
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+Built with Ruby on Rails 
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add your Categories for expenses
+- Add new transactions for each category
+- Get the total per category for each category
+- Can have many users to use the app
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://rails-budget-app-s2ql.onrender.com/)
+
```

Comment:
- [ ] when trying to add a transaction in the deployed application the user gets below error:
![afbeelding](https://user-images.githubusercontent.com/39240820/219474977-422a07bb-8f6d-43f7-b1b9-f5a9bad4e019.png)